examples/guestbook/README.md

```diff
@@ -436,8 +436,8 @@ If you are in a live kubernetes cluster, you can just kill the pods, using a scr
 
 ```shell
 ### First, kill services and controllers.
-kubectl stop rc examples/guestbook/redis-slave-controller.json
-kubectl stop rc examples/guestbook/frontend-controller.json
+kubectl stop rc examples/guestbook/redis-slave-controller
+kubectl stop rc examples/guestbook/frontend-controller
```

Comment:
It currently works because kubectl happens to strip off everything except whatever's behind the final slash. I'd prefer the `-f` version, but if you want to keep this form, could you at least strip off the "examples/guestbook/" part of the strings so that all that's left is the actual name of the replication controller, like this?
```
kubectl stop rc redis-slave-controller
kubectl stop rc frontend-controller
```